pkg/envinfo/envinfo.go

```diff
@@ -184,7 +196,7 @@ func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}
 
 		return esi.writeToFile()
 	}
-	return errors.Errorf("unknown parameter :'%s' is not a parameter in envinfo", parameter)
+	return errors.Errorf("unknown parameter: %s is not a parameter in the odo environment", parameter)
```

Comment:
Should be `in the odo environment file`

Reply:
Can we add something like `refer odo env set --help to see valid options`?

Reply:
Yes, we can add that.